### PR TITLE
playbook file

### PR DESCRIPTION
**Pull request by sjamgade**

### CODE REVIEW

File: playbook.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+`monasca metric-statistics`
+
+After some time there should some metrics for the VMs from stack.
+To get values other than 0 (zero), create some load on the VMs
+
+```
+  # Make sure to type in SERVER_ID on next line
+
+  export FLOATING_IP=`openstack floating ip create public -f value -c floating_ip_address --port=$(openstack port list -f value -c id --device-id=$(openstack server show -f value -c id SERVER_ID ))`
+  ssh cirros@$FLOATING_IP "dd if=/dev/zero of=/dev/null &"
```
Comment (witekest) [inline playbook.md]: Logging into the instance does not work in the OVH environment.

**Review by witekest — CHANGES_REQUESTED** [review]

File: playbook.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+**DO**:
+
+Fill out resource block for **cpu_alarm_high**, **cpu_alarm_low**, **group**:
+  - **OS::Heat::AutoScalingGroup**
+  - **OS::Monasca::AlarmDefinition**  
+  Add properties as defined in [heatintro.md](./heatintro.md)
+
+Create a stack  
+`openstack stack create --wait -t autoscaling.yaml test`
```
Comment (jodavis-suse) [inline playbook.md:14]: At line 14 we instruct them to call the stack 'test', but on line 21 we reference 'teststack'

File: autoscaling.yaml [hunk 3]
```diff
@@ -12,78 +12,53 @@ parameters:
...
+      desired_capacity: ##FILLHERE##
+      max_size: ##FILLHERE##
       min_size: 1
       resource:
         type: OS::Nova::Server
         properties:
           flavor: { get_param: flavor }
           image: { get_param: image }
-          metadata: {"scale_group": {get_param: "OS::stack_id"}}
+          metadata : {"scale_group": {get_param: "OS::stack_id"}}
```
Comment (jodavis-suse) [inline autoscaling.yaml:30]: extra space added here.

File: playbook.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+Check metadata attribute of server.   
+It should have `scale_group=(id of stack named teststack)`  
+`openstack server show SERVER_ID`
+
+There should be two alarm definitions created.  
+`monasca alarm-definition-list`
+
+Check if the metric used in alarm expression is available in the TSDB. Use
+`--help` to look up available options.   
+`monasca metric-list`
```
Comment (jodavis-suse) [inline playbook.md:29]: I get an empty list on my devstack at this 'monasca metric-list' command unless I change the rc file to use mini-mon as the user (or sign in another session which picks up the mini-mon env).

File: playbook.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+What are the actual values of the relevant metrics? You can use `--group_by`
+argument.  
+`monasca measurement-list`
```
Comment (jodavis-suse) [inline playbook.md:33]: Might give a hint about how to format the time.
date -Iseconds | sed -e "s/.\{6\}$/Z/"
or -120

File: playbook.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+configured scaling policies.
+
+## 3. Add scaling policies
+
+**DO**:
+
+Delete previous stack  
+`openstack stack delete --wait --yes teststack`
+
+- Fill out  *scale_up_policy* and *scale_up_policy*:
```
Comment (jodavis-suse) [inline playbook.md]: Second should be "scale_down_policy"

**Review by jodavis-suse — COMMENTED** [review]
Looking for things to clarify as I went through it.

**Comment by sjamgade** [comment]
Merging for now, @jodavis-suse I will address your comments in master

**Pull request merged**